content/applications/hr/employees/new_employee.rst

```diff
@@ -34,20 +42,20 @@ Required fields
 Optional fields
 ---------------
 
-- :guilabel:`Photo`: In the top right image box of the employee form, click on the :guilabel:`✏️
+- :guilabel:`Photo`: in the top right image box of the employee form, click on the :guilabel:`✏️
   (pencil)` edit icon to select a photo to upload.
-- :guilabel:`Job Position`: Enter the employee's job position title under their name, or select the
-  job position from the :guilabel:`Job Position` drop-down menu to have this field auto-populate.
-  The :guilabel:`Job Position` field under the employee name can be modified and does not need to
-  match the selection made in the :guilabel:`Job Position` drop-down.
+- :guilabel:`Job Position`: enter the employee's job title under their name or select it from the
+  :guilabel:`Job Position` drop-down menu to have this field auto-populate. The :guilabel:`Job
+  Position` field under the employee name can be modified and does not need to match the selection
+  made in the :guilabel:`Job Position` drop-down.
 
   .. example::
-     It is recommended to have the job positions match, but if desired, the typed-in description in
-     this field can contain more information than the selected drop-down job position.
+     While it is recommended to have the job positions match, the typed-in description in this field
+     can contain more information than the selected drop-down job position if desired.
 
-     An example where this may be applicable is if someone is hired for a sales representative
-     position configured as :guilabel:`Sales Representative` in the *Recruitment* app, and that is
-     selected for the drop-down :guilabel:`Job Position` field.
+     For instance, if someone is hired for a sales representative position configured as
+     :guilabel:`Sales Representative` in the Recruitment app, that can be selected in the drop-down
+     :guilabel:`Job Position`` field.
 
      In the typed in job position field beneath the employee's name, the position could be more
      specific, such as `Sales Representative - Subscriptions` if the employee is focused solely on
```

Comment:
I am a little confused since I see it in the green text in the suggestion block right above? I made the edits to match what I see in the green. If this is not what you meant, then please let me know.